config/initializers/devise.rb

```diff
@@ -12,7 +12,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+  config.mailer_sender = 'polly@fishoflaurel.org'
```

Comment:
Some services send emails from a shared email account, so that if someone responds, multiple people can get at that email instead of only a single person. If you've got an `info@` or `help@` email account, that might not be a bad idea.

Reply:
Ha!  Multiple people, that sounds awesome sign me up for that!!  😆 We don't have that many volunteers so all the inquiries come to me and I divy them out.  Unfortunately we have a very small core team of people who are actively daily involved.  Good idea though!!

Reply:
lol Cool deal. Just wanted to check. :-)

Reply:
Yeah of Course, thanks!  That would be ideal :)